src/servers/public-api/books/books.controller.ts

```diff
@@ -48,7 +49,16 @@ export class BooksController {
     @Next() next: ExpressNextFunction
   ) {
     try {
-      const book = await services.books.sortBook(request.body);
+      const page = serializePagination(request).page;
+      const limit = serializePagination(request).limit;
+      const startIndex = (page - 1) * limit;
+      const countData = await services.books.countBook(request.body.filter)
+      const book = await services.books.sortBook(
+        request.body.sort,
+        request.body.filter,
+        startIndex,
+        limit
+      );
```

Comment:
For this, the body in postman is something like this: 

{
  "sort": {
      "title": "desc"
  },
  "filter": {
      "author": "Safwan"
  }
}